lib/projectPreprocessor.js

```diff
@@ -190,6 +191,17 @@ class ProjectPreprocessor {
 	async loadProjectConfiguration(project) {
 		if (project.specVersion) { // Project might already be configured
 			// Currently, specVersion is the indicator for configured projects
+
+			// Validate project config, but exclude additional properties
+			const excludedProperties = ["id", "version", "path", "dependencies"];
+			const entries = Object.entries(project).filter(([key]) => {
+				return !excludedProperties.includes(key);
+			});
+			const config = Object.fromEntries(entries);
+			await validate({
+				config
+			});
```

Comment:
Maybe move this into a separate function? It only needs `project`, should be pretty easy

Reply:
Done